staging/src/k8s.io/client-go/pkg/apis/policy/v1alpha1/register.go

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
I did both of them together because staging/copy.sh is / was broken for a bit.  And both need to change at once, so I wanted to be clear that both sides are the same.